specs/AgriFood/AgriCrop/schema.json

```diff
@@ -0,0 +1,110 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/AgriCrop/schema.json",
+    "title": "FIWARE - Agri Food",
+    "description": "This entity contains a harmonised description of a generic crop. This entity is primarily associated with the agricultural vertical and related IoT applications.",
+    "type": "object",
```

Comment:
why don't you include common schemas from FIWARE Data Models? 

Please check other schemas

Reply:
Fixed in (https://github.com/Ficodes/dataModels/commit/b8969ba0ff3b70e9405c22ede6a35774a899d50d)